yarn.lock

```diff
@@ -13151,6 +13151,15 @@ react-native-screens@4.11.1, react-native-screens@~4.11.1:
     react-native-is-edge-to-edge "^1.1.7"
     warn-once "^0.1.0"
 
+react-native-screens@4.11.1-nightly-20250611-8b82e081e:
+  version "4.11.1-nightly-20250611-8b82e081e"
+  resolved "https://registry.yarnpkg.com/react-native-screens/-/react-native-screens-4.11.1-nightly-20250611-8b82e081e.tgz#fa41372cc3a31b217513758597b6a9f6a95cd43c"
+  integrity sha512-6hF155fY5z3uuBHG6qYNpu0XuqAKn5omNATzVCAwMPlpiIJ2j+iHOWbYVyqBT8bDnVvcczZT/eloGvt6A2OtEQ==
+  dependencies:
+    react-freeze "^1.0.0"
+    react-native-is-edge-to-edge "^1.1.7"
+    warn-once "^0.1.0"
+
```

Comment:
`screenId` prop inside native screen was added for this feature to work. Thus using nigthly version of screens for testing.

https://github.com/software-mansion/react-native-screens/pull/2967